shared/teams/role-picker/index.js

```diff
@@ -191,17 +196,17 @@ export const RoleConfirm = ({
           marginRight: globalMargins.small,
         }}
       >
-        <Text type="BodyBig">Make {username} a {selectedRole} of {teamname}?</Text>
+        <Text type="BodyBig">{prompt}</Text>
       </Box>
       <Box style={{...globalStyles.flexBoxRow, margin: globalMargins.tiny}}>
         <Text type="BodySemibold">{introText}</Text>
       </Box>
       <Box style={{...globalStyles.flexBoxColumn, width: 280}}>{cans}</Box>
-      {cannots.length > 0 &&
+      {cannots &&
```

Comment:
Don't you still want to check for length? what if it's an empty list? maybe above do `(permissions.cannot || []).map(...)` so you don't have to deal with a `?Array<*>` type (just a `Array<*>` type)